CLDConfig/CLDReconstruction.py

```diff
@@ -99,17 +102,16 @@
                           "HowOften": ["1"]
                           }
 
-# TODO: put this somewhere else, needs to be in front of the output for now :(
 # setup AIDA histogramming and add eventual background overlay
 algList.append(MyAIDAProcessor)
 sequenceLoader.load("Overlay/Overlay")
 # tracker hit digitisation
 sequenceLoader.load("Tracking/TrackingDigi")
 
 # tracking
-if CONFIG["Tracking"] == "Truth":
+if reco_args.truthTracking:
```

Comment:
```suggestion
if reco_args.truthTracking and reco_args.conformalTracking:
   raise ValueError("Cannot enable truth and conformal tracking at the same time")
if reco_args.truthTracking:
```

Reply:
In principle this check should not be needed because the arguments are in a mutually exclusive group. However, I don't think the TruthTracking can be turned on at all at the moment because it does not turn off the default enabled ConformalTracking. I will have another look and also add a test.

Reply:
Ahh, I missed that this feature is there and exists in the first place. Then this is fine!

Reply:
Ah and to navigate around the ConformalTracking default when TruthTracking should be used I had already added the `elif`